### PR TITLE
SUBMARINE-813. Add mock test to pysubmarine models.

### PR DESCRIPTION
**Pull request by jeff-901**
### What is this PR for?
Add mock test to new pysubmarine feature -- model management.

### What type of PR is it?
Improvement

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-813

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

File: submarine-sdk/pysubmarine/tests/models/test_model.py [hunk 1]
```diff
@@ -25,7 +27,7 @@
 # Temporarily skip these tests after the following is solved:
 # TODO: Setup cluster by helm in CI/CD to enable mlflow server connection
 # TODO: Set an cooldown time between each test case
-@pytest.mark.skip(reason="no way of currently testing this")
+# @pytest.mark.skip(reason="no way of currently testing this")
```
Comment (jojochuang) [inline submarine-sdk/pysubmarine/tests/models/test_model.py:30]: If this annotation no longer applies, please remove it entirely.

Comment (jeff-901) [reply submarine-sdk/pysubmarine/tests/models/test_model.py]: ok. I remove it.

**Review by xunliu — APPROVED** [review]
@jeff-901 Thanks you for contributons.
In addition, I saw python checks have failing, Please fixed it. Thanks.

File: submarine-sdk/pysubmarine/tests/models/test_model.py [hunk 2]
```diff
@@ -15,17 +15,19 @@
...
+import mlflow
 import numpy as np
 import pytest
+from mlflow.tracking import MlflowClient
 
 from pytorch import LinearNNModel
 from submarine import ModelsClient
 
-
 # Temporarily skip these tests after the following is solved:
```
Comment (ByronHsu) [inline submarine-sdk/pysubmarine/tests/models/test_model.py:26]: These three lines can be removed too

Comment (jeff-901) [reply submarine-sdk/pysubmarine/tests/models/test_model.py]: Thanks!

**Comment by ByronHsu** [comment]
LGTM

**Pull request merged**